Source/Engine/Physics/Actors/RigidBody.h

```diff
@@ -430,6 +430,12 @@ class FLAXENGINE_API RigidBody : public Actor, public IPhysicsActor
     /// <param name="result">The result point on the rigidbody shape that is closest to the specified location.</param>
     API_FUNCTION() void ClosestPoint(const Vector3& position, API_PARAM(Out) Vector3& result) const;
 
+    /// <summary>
+    /// Moves and rotates the rigidbody in world space within the limits of defined constraints.
+    /// </summary>
+    /// <param name="translation">The translation vector.</param>
+    /// <param name="rotation">The rotation quaternion.</param>
+    API_FUNCTION() void AddMovement(const Vector3& translation, const Quaternion& rotation) override;
```

Comment:
Move it down below `// [Actor]` and remove not needed XML doc comments and `API_FUNCTION()` so it's just pure override (base Actor class has proper docs and exposed scripting info).